src/bundle/Resources/public/js/scripts/helpers/tooltips.helper.js

```diff
@@ -1,5 +1,17 @@
-(function(global, doc, eZ, $) {
+(function (global, doc, eZ, $) {
     const TOOLTIPS_SELECTOR = '[title]';
+    const config = {
+        attributes: true,
+        childList: true,
+        subtree: true,
+    };
+    const observer = new MutationObserver((mutationsList, observer) => {
```

Comment:
Unnecessary second arg `observer` (could be even misleading, as variable `observer` is already defined)